_posts/2023-05-15-May-Update-Finishing-my-Second-Igalia-CE.md

```diff
@@ -0,0 +1,297 @@
+So, on the final part of my Igalia CE, I inspected a couple of IGT failures and sent fixes to address the errors.
+
+## Linux Kernel
+
+This patch is a revival of a series I sent in January to fix the IGT test `kms_addfb_basic@addfb25-bad-modifier`.
+This test was also failing in VC4 and I investigated the reason back in January.
+I sent a patch to guarantee that the test would pass and after some feedback, I came down to a dead end.
+So, I left this patch aside for a while and decided to recapture it now.
+Now, with this patch being merged, we can guarantee that the test  `kms_addfb_basic@addfb25-bad-modifier` is passing for multiple drivers.
+
+| Patches | Status |
+| :---- | :----: |
+| [[PATCH] drm/gem: Check for valid formats](https://lore.kernel.org/dri-devel/20230412142923.136707-1-mcanal@igalia.com/T/) | **Accepted** |
+
+## IGT
+
+On the IGT side, I sent a couple of improvements to the tests.
+The failure was usually just a scenario that the test didn't consider.
+For example, the `kms_plane_scaling` test was failing in VKMS, because it didn't consider the case in which the driver did not have the rotation property.
+As VKMS didn't use to have the rotation property, the tests were failing instead of skipping.
+Therefore, I just developed a path for drivers without the rotation property for the tests to skip.
+
+I sent improvements to the `kms_plane_scaling`, `kms_flip`, and `kms_plane` tests, making the tests pass or skip on all cases for the VKMS.
+
+| Patches | Status |
+| :---- | :----: |
+| [[PATCH i-g-t] tests/kms_plane_scaling: negative tests can return -EINVAL or -ERANGE](https://patchwork.freedesktop.org/patch/533643/?series=116907&rev=1) | **Accepted** |
+| [[PATCH i-g-t] tests/kms_plane_scaling: fix variable misspelling](https://patchwork.freedesktop.org/patch/533645/?series=116907&rev=1)                     | **Accepted** |
+| [[PATCH i-g-t] tests/kms_plane_scaling: remove unused parameters](https://patchwork.freedesktop.org/patch/533644/?series=116907&rev=1)                     | **Accepted** |
+| [[PATCH i-g-t] tests/kms_plane_scaling: Only set rotation if rotation != rotate-0](https://patchwork.freedesktop.org/series/117250/)                       | **Accepted** |
+| [[PATCH v2 i-g-t] tests/kms_flip: Check if is Intel device before doing all the setup](https://patchwork.freedesktop.org/series/117246/)                   | **Accepted** |
+| [[PATCH i-g-t v2] tests/kms_plane: allow pixel-format tests to run on drivers without legacy LUT](https://patchwork.freedesktop.org/patch/533753/)         | In Review |
+
+
+## VKMS CI List
+
+One important thing to VKMS is creating a baseline of tests that should pass.
+This way new contributions can be tested against this baseline and we can avoid introducing regressions in the codebase.
+This is why I sent a patch to IGT to create a testlist for the VKMS driver.
+This is a list of all the KMS tests that must pass on the VKMS driver.
```

Comment:
I sent a patch to IGT to create a testlist for the VKMS driver with all the KMS tests that must pass on the VKMS driver.

NOTE: say it's good for maintenance